rules/linux/file_event/file_event_lnx_persistence_cron_files.yml

```
@@ -1,12 +1,13 @@
 title: Persistence Via Cron Files
```

For a more accurate title. Since we only see the creation not the contents
```suggestion
title: New Cron File Created
```